src/PointwiseFunctions/Xcts/AdmMass.cpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/Xcts/AdmMass.hpp"
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Slice.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace Xcts {
+
+void adm_mass_surface_integrand(
+    gsl::not_null<tnsr::I<DataVector, 3>*> result,
+    const tnsr::i<DataVector, 3>& deriv_conformal_factor,
+    const tnsr::II<DataVector, 3>& inv_conformal_metric,
+    const tnsr::Ijj<DataVector, 3>& conformal_christoffel_second_kind) {
+  tenex::evaluate<ti::I>(
+      result, 1. / (16. * M_PI) *
+                  (inv_conformal_metric(ti::J, ti::K) *
+                       conformal_christoffel_second_kind(ti::I, ti::j, ti::k) -
+                   inv_conformal_metric(ti::I, ti::J) *
+                       conformal_christoffel_second_kind(ti::K, ti::j, ti::k) -
```

Comment:
$\Gamma^k_{jk}$ is available from the DataBox as `Xcts::Tags::ConformalChristoffelContracted`, please pass it in as an argument here.